AdminOpsGuide/dbrepl.rst

```diff
@@ -1191,141 +1193,312 @@ Here is the code:
 Setting up an A -> B Replication Configuration with Empty Databases
```

Comment:
I would like each setup example to show up as a hyperlink under the "Database Replication" link in the top level document.

Reply:
Done

Reply:
Thanks. The document flows much better now. I don't have any other comments.